tests/setup-tests.sh

```diff
@@ -42,6 +42,12 @@ FreeBSD)
     sysctl -w net.link.tap.up_on_open=1
     ifconfig tap100 create 10.0.0.1/24 link0 up
     ;;
+OpenBSD)
+     cd /dev
+     ./MAKEDEV tap100
```

Comment:
Better to wrap this in `()`'s, so we come back to the current dir (just in case we add something at the end of the script that has to be at curr dir).